releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -149,10 +149,21 @@
 	</location>
 	
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		<repository location="http://download.eclipse.org/tools/orbit/R-builds/R20170516192513/repository/"/>
+		<repository location="http://download.eclipse.org/tools/orbit/downloads/drops/S20180119201206/repository/"/>
```

Comment:
Is this a stable URL?

Reply:
how to find stable urls?

Reply:
Looks like the S...-drop is a milestone, so that should be ok.

Reply:
ok

Reply:
ASM6 is not yet available in a release repo. Stable repos can be used for a while, at least until after Photon release. When the Photon Orbit is available as a release, then this would have to change again.
I think it is OK to use that repo now. We may open a reminder issue to change that later.

Reply:
we changed this several times over the recent releases. i dont know why orbit did not ship a stable release for oxygen.2